src/ProjectNeon/Assets/Z_Tests/Features/Effects/EvadeTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+using NUnit.Framework;
+using UnityEngine;
+
+public sealed class EvadeTests
```

Comment:
We need to test to ensure that the second attack is NOT evaded.

Reply:
@SilasReinagel Actually these tests are about the `Evade` effect, not the `Recurrent`. I've refactored them to use only the `Evade` implementation